docs/websocket/feedback.md

```diff
@@ -123,6 +159,16 @@ id   | `string` | **Required** The id of the feedback to pause.
 
 A paused feedback can be resumed using this RPC. After this the audio stream is
 again processed by the server.
+Currently **it is required to re-send the audio header when resuming
+feedback**. Failing to do so will fail the feedback as the server can't
+recognise the audio format without a header.
+
+!!! note
+    It appears to be valid to have a wave file with multiple headers.
+
+!!! warning
+    It is required to re-send the audio header upon resume. The audio
+    processing will fail if the header isn't re-sent.
```

Comment:
Is this extra note necessary since you already put the message **it is required to re-send the audio header when resuming feedback** before?

Reply:
Shouldn't be, but I thought it would be best to emphasize it since I recon people will be reading over it.

Reply:
It is not necessary to have this redundant comment, please remove it.